docs/reference/search/search.asciidoc

```diff
@@ -4,18 +4,22 @@
 Returns search hits that match the query defined in the request.
 
 [source,console]
---------------------------------------------------
-GET /twitter/_search?q=user:kimchy
```

Comment:
This example has been moved here:
https://github.com/elastic/elasticsearch/pull/47042/files#diff-bcfac6633e675424434326d14790f815R326